generic3g/vertical/Test_VerticalLinearMap.F90

```diff
@@ -0,0 +1,40 @@
+#define I_AM_MAIN
+#include "MAPL_Generic.h"
```

Comment:
This looks like a duplicate of the file above.  And should be deleted.

Reply:
This is indeed a duplicate, and is used to quickly run the test, instead of having to go through the `ctest` framework.

I'll remove it once